tor_archivist/core/helpers.py

```diff
@@ -87,12 +86,11 @@ def signal_handler(signal, frame):
     global running
 
     if not running:
-        logging.critical('User pressed CTRL+C twice!!! Killing!')
+        logging.critical("User pressed CTRL+C twice!!! Killing!")
         sys.exit(1)
 
     logging.info(
-        '\rUser triggered command line shutdown. Will terminate after current '
-        'loop.'
+        "\rUser triggered command line shutdown. Will terminate after current " "loop."
```

Comment:
can we remove the extra quotes in this line

Reply:
We sure can!